keyboards/keychron/q1/rev_0102/info.json

```diff
@@ -0,0 +1,93 @@
+	    {"label":"F9", "x":9.75, "y":0},
+	    {"label":"F10", "x":10.75, "y":0}, 
+	    {"label":"F11", "x":11.75, "y":0}, 
+	    {"label":"F12", "x":12.75, "y":0}, 
+	    {"label":"Delete", "x":14, "y":0}, 
+	    {"label":"Insert", "x":15.25, "y":0}, 
+	    {"label":"~", "x":0, "y":1},
+	    {"label":"!", "x":1, "y":1}, 
+	    {"label":"@", "x":2, "y":1}, 
+	    {"label":"#", "x":3, "y":1}, 
+	    {"label":"$", "x":4, "y":1}, 
+	    {"label":"%", "x":5, "y":1}, 
+	    {"label":"^", "x":6, "y":1}, 
+	    {"label":"&", "x":7, "y":1}, 
+	    {"label":"*", "x":8, "y":1},
+	    {"label":"(", "x":9, "y":1}, 
+	    {"label":")", "x":10, "y":1},
+	    {"label":"_", "x":11, "y":1}, 
+	    {"label":"+", "x":12, "y":1}, 
+	    {"label":"Backspace", "x":13, "y":1, "w":2}, 
+	    {"label":"Page Up", "x":15.25, "y":1}, 
+	    {"label":"Tab", "x":0, "y":2, "w":1.5},
+	    {"label":"Q", "x":1.5, "y":2},
+	    {"label":"W", "x":2.5, "y":2}, 
+	    {"label":"E", "x":3.5, "y":2}, 
+	    {"label":"R", "x":4.5, "y":2}, 
+	    {"label":"T", "x":5.5, "y":2}, 
+	    {"label":"Y", "x":6.5, "y":2}, 
+	    {"label":"U", "x":7.5, "y":2}, 
+	    {"label":"I", "x":8.5, "y":2}, 
+	    {"label":"O", "x":9.5, "y":2}, 
+	    {"label":"P", "x":10.5, "y":2}, 
+	    {"label":"{", "x":11.5, "y":2}, 
+	    {"label":"}", "x":12.5, "y":2},
+	    {"label":"|", "x":13.5, "y":2, "w":1.5}, 
+	    {"label":"Page Down", "x":15.25, "y":2}, 
+	    {"label":"Caps Lock", "x":0, "y":3, "w":1.75}, 
+	    {"label":"A", "x":1.75, "y":3}, 
+	    {"label":"S", "x":2.75, "y":3},
+	    {"label":"D", "x":3.75, "y":3}, 
+	    {"label":"F", "x":4.75, "y":3}, 
+	    {"label":"G", "x":5.75, "y":3}, 
+	    {"label":"H", "x":6.75, "y":3}, 
+	    {"label":"J", "x":7.75, "y":3}, 
+	    {"label":"K", "x":8.75, "y":3}, 
+	    {"label":"L", "x":9.75, "y":3}, 
+	    {"label":":", "x":10.75, "y":3},
+	    {"label":"\"", "x":11.75, "y":3}, 
+	    {"label":"Enter", "x":12.75, "y":3, "w":2.25}, 
+	    {"label":"Home", "x":15.25, "y":3}, 
+	    {"label":"Shift", "x":0, "y":4, "w":2.25}, 
+	    {"label":"Z", "x":2.25, "y":4}, 
+	    {"label":"X", "x":3.25, "y":4}, 
+	    {"label":"C", "x":4.25, "y":4}, 
+	    {"label":"V", "x":5.25, "y":4}, 
+	    {"label":"B", "x":6.25, "y":4}, 
+	    {"label":"N", "x":7.25, "y":4}, 
+	    {"label":"M", "x":8.25, "y":4},
+	    {"label":"<", "x":9.25, "y":4}, 
+	    {"label":">", "x":10.25, "y":4}, 
+	    {"label":"?", "x":11.25, "y":4}, 
+	    {"label":"Shift", "x":12.25, "y":4, "w":1.75},
+	    {"label":"\u2191", "x":14.25, "y":4.25}, 
+	    {"label":"Ctrl", "x":0, "y":5, "w":1.25}, 
+	    {"label":"Win", "x":1.25, "y":5, "w":1.25},
+	    {"label":"Alt", "x":2.5, "y":5, "w":1.25}, 
+	    {"x":3.75, "y":5, "w":6.25}, 
+	    {"label":"Alt", "x":10, "y":5}, 
+	    {"label":"Fn", "x":11, "y":5}, 
+	    {"label":"Ctrl", "x":12, "y":5}, 
+	    {"label":"\u2190", "x":13.25, "y":5.25}, 
+	    {"label":"\u2193", "x":14.25, "y":5.25}, 
+	    {"label":"\u2192", "x":15.25, "y":5.25}]
```

Comment:
* applies four-space indent
* adds line breaks between each row of the layout
* adds 0.25u gap between Function row and Number row
* deletes ANSI Backslash object
* adds objects for Non-US Hash key and Non-US Backslash (`#~` and `\|` in ISO-UK, respectively)

```suggestion
            "layout": [
                {"label":"Esc", "x":0, "y":0},
                {"label":"F1", "x":1.25, "y":0},
                {"label":"F2", "x":2.25, "y":0},
                {"label":"F3", "x":3.25, "y":0},
                {"label":"F4", "x":4.25, "y":0},
                {"label":"F5", "x":5.5, "y":0},
                {"label":"F6", "x":6.5, "y":0},
                {"label":"F7", "x":7.5, "y":0},
                {"label":"F8", "x":8.5, "y":0},
                {"label":"F9", "x":9.75, "y":0},
                {"label":"F10", "x":10.75, "y":0},
                {"label":"F11", "x":11.75, "y":0},
                {"label":"F12", "x":12.75, "y":0},
                {"label":"Delete", "x":14, "y":0},
                {"label":"Insert", "x":15.25, "y":0},

                {"label":"~", "x":0, "y":1.25},
                {"label":"!", "x":1, "y":1.25},
                {"label":"@", "x":2, "y":1.25},
                {"label":"#", "x":3, "y":1.25},
                {"label":"$", "x":4, "y":1.25},
                {"label":"%", "x":5, "y":1.25},
                {"label":"^", "x":6, "y":1.25},
                {"label":"&", "x":7, "y":1.25},
                {"label":"*", "x":8, "y":1.25},
                {"label":"(", "x":9, "y":1.25},
                {"label":")", "x":10, "y":1.25},
                {"label":"_", "x":11, "y":1.25},
                {"label":"+", "x":12, "y":1.25},
                {"label":"Backspace", "x":13, "y":1.25, "w":2},
                {"label":"Page Up", "x":15.25, "y":1.25},

                {"label":"Tab", "x":0, "y":2.25, "w":1.5},
                {"label":"Q", "x":1.5, "y":2.25},
                {"label":"W", "x":2.5, "y":2.25},
                {"label":"E", "x":3.5, "y":2.25},
                {"label":"R", "x":4.5, "y":2.25},
                {"label":"T", "x":5.5, "y":2.25},
                {"label":"Y", "x":6.5, "y":2.25},
                {"label":"U", "x":7.5, "y":2.25},
                {"label":"I", "x":8.5, "y":2.25},
                {"label":"O", "x":9.5, "y":2.25},
                {"label":"P", "x":10.5, "y":2.25},
                {"label":"{", "x":11.5, "y":2.25},
                {"label":"}", "x":12.5, "y":2.25},
                {"label":"Page Down", "x":15.25, "y":2.25},

                {"label":"Caps Lock", "x":0, "y":3.25, "w":1.75},
                {"label":"A", "x":1.75, "y":3.25},
                {"label":"S", "x":2.75, "y":3.25},
                {"label":"D", "x":3.75, "y":3.25},
                {"label":"F", "x":4.75, "y":3.25},
                {"label":"G", "x":5.75, "y":3.25},
                {"label":"H", "x":6.75, "y":3.25},
                {"label":"J", "x":7.75, "y":3.25},
                {"label":"K", "x":8.75, "y":3.25},
                {"label":"L", "x":9.75, "y":3.25},
                {"label":":", "x":10.75, "y":3.25},
                {"label":"\"", "x":11.75, "y":3.25},
                {"label":"#~", "x":12.75, "y":3.25},
                {"label":"Enter", "x":13.75, "y":2.25, "w":1.25, "h":2},
                {"label":"Home", "x":15.25, "y":3.25},

                {"label":"Shift", "x":0, "y":4.25, "w":1.25},
                {"label":"\\|", "x":1.25, "y":4.25},
                {"label":"Z", "x":2.25, "y":4.25},
                {"label":"X", "x":3.25, "y":4.25},
                {"label":"C", "x":4.25, "y":4.25},
                {"label":"V", "x":5.25, "y":4.25},
                {"label":"B", "x":6.25, "y":4.25},
                {"label":"N", "x":7.25, "y":4.25},
                {"label":"M", "x":8.25, "y":4.25},
                {"label":"<", "x":9.25, "y":4.25},
                {"label":">", "x":10.25, "y":4.25},
                {"label":"?", "x":11.25, "y":4.25},
                {"label":"Shift", "x":12.25, "y":4.25, "w":1.75},
                {"label":"\u2191", "x":14.25, "y":4.5},

                {"label":"Ctrl", "x":0, "y":5.25, "w":1.25},
                {"label":"Win", "x":1.25, "y":5.25, "w":1.25},
                {"label":"Alt", "x":2.5, "y":5.25, "w":1.25},
                {"label":"Space", "x":3.75, "y":5.25, "w":6.25},
                {"label":"Alt", "x":10, "y":5.25},
                {"label":"Fn", "x":11, "y":5.25},
                {"label":"Ctrl", "x":12, "y":5.25},
                {"label":"\u2190", "x":13.25, "y":5.5},
                {"label":"\u2193", "x":14.25, "y":5.5},
                {"label":"\u2192", "x":15.25, "y":5.5}
            ]
```